WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -728,33 +784,24 @@ private extension String {
     static let deleteButtonText = NSLocalizedString("Delete Permanently", comment: "Title for button on the comment details page that deletes the comment when tapped.")
 }
 
-
-// MARK: - CommentModerationBarDelegate
-
-extension CommentDetailViewController: CommentModerationBarDelegate {
-    func statusChangedTo(_ commentStatus: CommentStatusType) {
-
-        notifyDelegateCommentModerated()
-
-        switch commentStatus {
+private extension CommentStatusType {
+    var title: String? {
+        switch self {
         case .pending:
-            unapproveComment()
+            return NSLocalizedString("Pending", comment: "Button title for Pending comment state.")
         case .approved:
-            approveComment()
+            return NSLocalizedString("Approved", comment: "Button title for Approved comment state.")
         case .spam:
-            spamComment()
-        case .unapproved:
-            trashComment()
+            return NSLocalizedString("Spam", comment: "Button title for Spam comment state.")
```

Comment:
I _thought_ there was a P2 somewhere saying we needed to start adding keys to localized strings but all I could find was paNNhX-nP-p2. It doesn't say we need keys, just that the option is there.

Regardless, I don't think it's a huge deal since I can't find the post anywhere. Just wanted to point it out in case you happen to know more.

Reply:
Oh no I saw that as well and have been doing it. These are existing strings but I can do it regardless